### PR TITLE
add ANALYZE statement support

### PR DESCRIPTION
**Pull request by michity15169**
T/O

https://cloud.google.com/spanner/docs/reference/standard-sql/data-definition-language#analyze-statistics

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,6 +177,7 @@ and `{}` for a mutually exclusive keyword.
 | Show DML Execution Plan | `EXPLAIN {INSERT\|UPDATE\|DELETE} ...;` | |
 | Show Query Execution Plan with Stats | `EXPLAIN ANALYZE SELECT ...;` | |
 | Show DML Execution Plan with Stats | `EXPLAIN ANALYZE {INSERT\|UPDATE\|DELETE} ...;` | |
+| Start a new query optimizer statistics package construction | `ANALYZE` | |
```
Comment (yfuruyama) [inline README.md:180]: nit: To keep consistent with other statements, please add `;` in the end: `ANALYZE;`

**Review by yfuruyama — COMMENTED** [review]
Thank you for implementing this new statement. I left a minor comment, but overall LGTM.

File: README.md (hunk 1 above)
Comment (michity15169) [reply README.md]: Your point sure makes sense. I have updated Syntax doc.

Comment (yfuruyama) [reply README.md]: Thanks!

**Review by yfuruyama — APPROVED** [review]
LGTM

**Pull request merged**